docs/ossm/kiali-multi-cluster/README.md

```diff
@@ -0,0 +1,137 @@
+[Return to OSSM Docs](../)
+
+# Kiali - multi-cluster
+
+For multi-cluster Istio deployments, Kiali can show you a single unified view of your mesh across clusters.
+
+Before proceeding with the setup, ensure you meet the requirements.
+
+## Requirements
+
+- Two openshift clusters. In this tutorial they are named `east` and `west`.
+- Istio installed in a multi-cluster configuration on each cluster.
+- IstioCNI installed on each cluster.
+- Aggregated metrics and traces. Kiali needs a single endpoint for metrics and a single endpoint for traces where it can consume aggregated metrics/traces across all clusters. There are multiple ways to aggregate metrics/traces such as Prometheus federation or using OTEL collector pipelines.
+- Cluster Admin for each cluster.
+- Kiali Operator v1.89 installed on the `east` cluster.
+
+## Setup
+
+In this tutorial, we will deploy Kiali on the `east` cluster and then grant Kiali access to the `west` cluster. The unified multi-cluster setup requires the Kiali Service Account (SA) to have read access to each Kubernetes cluster in the mesh. This is separate from the user credentials that are required when a user logs into Kiali. Kiali uses the user's credentials to check if the user has access to a namespace and when performing any write operation such as creating/editing/deleting objects in Kubernetes. To give the Kiali Service Account access to each remote cluster, a kubeconfig with credentials needs to be created and mounted into the Kiali pod.
+
+### Procedure
+
+1. Install Kiali on the `east` cluster.
+
+   Create a file named `kiali.yaml`.
+
+   ```yaml
+   apiVersion: kiali.io/v1alpha1
+   kind: Kiali
+   metadata:
+     name: kiali
+     namespace: istio-system
+   spec:
+     version: v1.89
+   ```
+
+   Apply the yaml file into the `east` cluster.
+
+   ```sh
+   oc --context east apply -f kiali.yaml
+   ```
+
+   Wait for Kiali to become ready.
+
+   ```sh
+   oc --context east wait --for=condition=Successful --timeout=60s kialis/kiali -n istio-system
+   ```
+
+1. Create an `OAuthClient` on the remote cluster so that Kiali can access the OpenShift API server on behalf of users.
+
+   Find your Kiali URL
```

Comment:
This isn't really finding the URL, is it? "spec.host" I think is the route hostname (i.e. it doesn't include the protocol "http" or port number here).
```suggestion
   Find your Kiali route's hostname
```